go/cmd/dolt/commands/engine/sql_print.go

```diff
@@ -145,7 +146,7 @@ func prettyPrintResultsWithSummary(ctx *sql.Context, resultFormat PrintResultFor
 		if showWarnings {
 			warnings += "\n"
 			for _, warn := range ctx.Session.Warnings() {
-				warnings += fmt.Sprintf("\nWarning (Code %d): %s", warn.Code, warn.Message)
+				warnings += fmt.Sprintf(color.YellowString("\nWarning (Code %d): %s"), warn.Code, warn.Message)
```

Comment:
I would do this outside of Sprintf. like:
```suggestion
				warnings += color.YellowString(fmt.Sprintf("\nWarning (Code %d): %s"), warn.Code, warn.Message)
```